website/server/models/user/hooks.js

```diff
@@ -59,6 +66,10 @@ function _populateDefaultTasks (user, taskTypes) {
         });
       }
 
+      if (taskDefaults.tags) {
+        newTask.tags = _.map(taskDefaults.tags, findTag);
```

Comment:
Heads up, we're migrating to lodash v4 (https://github.com/HabitRPG/habitica/pull/8495) which includes many breaking changes (https://github.com/lodash/lodash/wiki/Changelog#v400). Before merging this PR should be checked and updated to make sure all lodash methods are compatible with v4.